scripts/update-readme.js

```diff
@@ -1,40 +1,35 @@
-import { readFileSync, writeFileSync } from 'node:fs';
+import { readdirSync, readFileSync, writeFileSync } from 'node:fs';
 import { fileURLToPath } from 'node:url';
 import { execSync } from 'node:child_process';
-import { parse, stringify } from 'yaml';
-
-const workflowFile = new URL('../.github/workflows/test-ecosystem.yml', import.meta.url);
-const workflowContent = readFileSync(workflowFile, 'utf8');
-const packages = parse(workflowContent).jobs.test.strategy.matrix.package;
-
-const failedPackages = new Set([
-	'@shopify/stylelint-plugin',
-	'@wordpress/stylelint-config',
-	'stylelint-codeframe-formatter',
-	'stylelint-codeframe-formatter',
-	'stylelint-config-rational-order',
-	'stylelint-config-recommended-less',
-	'stylelint-config-standard-vue',
-	'stylelint-config-tailwindcss',
-	'stylelint-config-wikimedia',
-	'stylelint-csstree-validator',
-	'stylelint-declaration-strict-value',
-	'stylelint-suitcss',
-	'stylelint-use-nesting',
-	'@stylistic/stylelint-config',
-]);
+import { parse } from 'yaml';
+import createStatusBadge from './utils/status-badge.js';
+import createNpmBadge from './utils/npm-badge.js';
+
+const workflowsDir = new URL('../.github/workflows', import.meta.url);
+let numberOfPackages = 0;
 
 const newPackageLines = [];
-newPackageLines.push('| Package | Status |');
-newPackageLines.push('| :------ | :----: |');
+newPackageLines.push('| Package | Latest Stylelint | Next Stylelint |');
+newPackageLines.push('| :------ | ----: |  ----: |');
```

Comment:
[ask] Just curious: why are the two columns for results on the right aligned, instead of left aligned? 🤔

Reply:
They were centered before and that looked odd with the badges (or at least it looked odd to me :) )

I chose right aligned because then the `failing` and `passing` (almost) line up.
This is a bit less noisy visually.

But maybe this also makes it harder to read the package names?

Happy to change it left aligned if you prefer :)

<img width="890" alt="Screenshot 2024-07-28 at 18 05 04" src="https://github.com/user-attachments/assets/f00fcf4a-4350-4759-93e9-6041c9472b03">
<img width="923" alt="Screenshot 2024-07-28 at 18 05 33" src="https://github.com/user-attachments/assets/29049a63-6a30-45a1-be33-78b07ec8769b">

Reply:
I got it. Let's keep it as-is. 👌🏼